### PR TITLE
Added files via upload

### PR DESCRIPTION
**Pull request by farzanaisrat**

### CODE REVIEW

File: MoodQuizStyle.css [hunk 1]
```diff
@@ -0,0 +1,368 @@
+h1 {
```
Comment (Taraque-A) [inline MoodQuizStyle.css:2]: Looks good!

File: MoodQuizStyle.css [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+	left: 0;
+	padding-top: 50px;
+	position: fixed;
+	right: 0;
+	top: 0;
+	z-index: -1;
+}
+
+
+@-webkit-keyframes animateCloud {
```
Comment (Taraque-A) [inline MoodQuizStyle.css:24]: Nice!

**Review by joerossi215 — APPROVED** [review]
Looks great! good job

File: MoodQuizStyle.css [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+        margin-left: -1000px;
+    }
+    100% {
+        margin-left: 100%;
+    }
+}
+
+/* ANIMATIONS */
+
+.x1 {
```
Comment (Taraque-A) [inline MoodQuizStyle.css:53]: This is looks awesome!

File: MoodQuizStyle.css [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+	animation: animateCloud 25s linear infinite;
+	
+	-webkit-transform: scale(0.55);
+	-moz-transform: scale(0.55);
+	transform: scale(0.55);
+    opacity: .4;
+}
+
+
+.cloud {
```
Comment (Taraque-A) [inline MoodQuizStyle.css:110]: Really nice!

File: MoodQuizStyle.css [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+.container {
+    background:#b5e2c3;
+    animation: fadeIn 2s;
+    -webkit-animation: fadeIn 2s;
+    -moz-animation: fadeIn 2s;
+    -o-animation: fadeIn 2s;
+    -ms-animation: fadeIn 2s;
+}
+
+@keyframes fadeIn {
```
Comment (Taraque-A) [inline MoodQuizStyle.css:183]: This is nice!

File: about.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+const textWrapper = document.querySelector('.rise');
+  textWrapper.innerHTML = textWrapper.textContent.replace(/\S/g, "<span class='letter'>$&</span>");
```
Comment (Taraque-A) [inline about.js:2]: Great work!

File: links.css [hunk 7]
```diff
@@ -81,10 +253,11 @@ h2 {
...
 li {
     list-style-type: none;
-    text-align: left;
-    font-size: 18px;
+    text-align: center;
 }
 
+
+
 a:link {
```
Comment (Taraque-A) [inline links.css:261]: Great idea!

File: links.js [hunk 8]
```diff
@@ -1,14 +1,73 @@
+function CollapsibleFactory() {}
```
Comment (Taraque-A) [inline links.js]: This was a great idea!

File: theme.css [hunk 9]
```diff
@@ -0,0 +1,6105 @@
...
+@keyframes p-component-overlay-leave-animation {
+  from {
+    background-color: var(--maskbg);
+  }
+  to {
+    background-color: transparent;
+  }
+}
+
+:root {
```
Comment (Taraque-A) [inline theme.css:117]: Nice!

**Review by Taraque-A — APPROVED** [review]

**Review by yellow-tulips — APPROVED** [review]
All of the files look good

**Review by tasnhubaislam — APPROVED** [review]
Looks great

**Pull request merged**